packages/webpack-cli/lib/utils/cli-flags.js

```diff
@@ -269,12 +275,13 @@ const coreFlags = cli
           }
 
           const inBuiltIn = builtInFlags.find((builtInFlag) => builtInFlag.name === flag);
+          const legacyMetadata = flagsForWebpack4[flag] || {};
 
           if (inBuiltIn) {
-              return { ...meta, name: flag, group: 'core', ...inBuiltIn };
+              return { ...meta, name: flag, group: 'core', ...inBuiltIn, ...legacyMetadata };
```

Comment:
I don't think it is right, why we need double `...`?

Reply:
yep redundant, removed